src/Dev/DevBuildController.php

```diff
@@ -17,7 +19,7 @@ class DevBuildController extends Controller
         'build'
     ];
 
-    public function build($request)
+    public function build(HTTPRequest $request): HTTPResponse
```

Comment:
For the benefit of the conversation in the parent issue:
This is a scenario where an `HTTPRequest` and `HTTPResponse` is explicitly being used in a CLI context from a controller action.